pkg/scheduler/framework/plugins/nodevolumelimits/csi_manager.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodevolumelimits
+
+import (
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	storagelisters "k8s.io/client-go/listers/storage/v1"
+	fwk "k8s.io/kube-scheduler/framework"
+)
+
+// CSIManagerImpl is a implementation of the CSIManager interface.
+type DefaultCSIManager struct {
+	defaultCSINodeLister *csiNodeLister
```

Comment:
Maybe without default (the struct is named default anyway)
```suggestion
	csiNodeLister *csiNodeLister
```

Reply:
I think it's generally ok to have the same names, as they won't conflict anyway, but it's up to you

Reply:
yeah I know, I am just stickler for that convention. lets leave it as it is.